deployment/scripts/install.sh

```diff
@@ -289,19 +299,50 @@ oc delete dc,rc,pod --selector logging-infra=kibana
 oc delete dc,rc,pod,daemonset --selector logging-infra=fluentd
 oc delete dc,rc,pod --selector logging-infra=elasticsearch
 
-for ((n=0;n<${es_cluster_size};n++)); do
-	oc process logging-es-template | oc create -f -
+declare -A pvcs=()
+for pvc in $(oc get persistentvolumeclaim --template='{{range .items}}{{.metadata.name}} {{end}}' 2>/dev/null); do
+  pvcs["$pvc"]=1  # note, map all that exist, not just ones labeled as supporting
+done
+for ((n=1;n<=${es_cluster_size};n++)); do
+        pvc="${ES_PVC_PREFIX}$n"
+        if [ "${pvcs[$pvc]}" != 1 -a "${ES_PVC_SIZE}" != "" ]; then # doesn't exist, create it
+          oc process logging-pvc-template -v "NAME=$pvc,SIZE=${ES_PVC_SIZE}" | oc create -f -
+          pvcs["$pvc"]=1
+        fi
+        if [ "${pvcs[$pvc]}" = 1 ]; then # exists (now), attach it
```

Comment:
No, assume that everything is OK for now or will resolve itself eventually (PVC unbound is fairly discoverable). Once we get around to adding validation, status of PVCs should be checked after deployment (metrics validation PR includes this already).